AnagramSolver.Models/Models/Words.cs

```diff
@@ -1,14 +1,8 @@
-﻿using System;
-using System.Collections.Generic;
-using System.Linq;
-using System.Text;
-using System.Threading.Tasks;
-
-namespace AnagramSolver.Models.Models
+﻿namespace AnagramSolver.Models.Models
 {
     public class Words
```

Comment:
At the moment this class only contains one string, which means this class can be removed and the code can be refactored (after tests cover the area you refactor)